ecs-cli/modules/config/readwriter_v1_test.go

```diff
@@ -239,36 +171,30 @@ compose-service-name-prefix = ecscompose-service-
 cfn-stack-name-prefix = amazon-ecs-cli-setup-
 `
 
-dest, err := newMockDestination()
-assert.NoError(t, err, "Error creating mock config destination")
-
-err = os.MkdirAll(dest.Path, *dest.Mode)
-assert.NoError(t, err, "Could not create config directory")
-
-defer os.RemoveAll(dest.Path)
+	dest, err := newMockDestination()
+	assert.NoError(t, err, "Error creating mock config destination")
 
-// Save old ini config file
-err = ioutil.WriteFile(dest.Path+"/"+configFileName, []byte(configContents), *dest.Mode)
-assert.NoError(t, err)
+	err = os.MkdirAll(dest.Path, *dest.Mode)
+	assert.NoError(t, err, "Could not create config directory")
 
-// Overwrite
-parser := setupParser(t, dest, false)
-saveConfigWithCluster(t, parser, dest)
+	defer os.RemoveAll(dest.Path)
 
+	// Save old ini config file
+	err = ioutil.WriteFile(dest.Path+"/"+iniConfigFileName, []byte(configContents), *dest.Mode)
+	assert.NoError(t, err)
 
-// Ensure that what has been read is correct
-readConfig, configMap, err := parser.GetConfig()
-assert.NoError(t, err, "Error reading config")
-assert.Equal(t, testClusterName, readConfig.Cluster, "Cluster name mismatch in config.")
-_, ok := configMap[composeProjectNamePrefixKey]
-assert.True(t, ok, "Compose project prefix name should exist in config.")
-assert.Empty(t, readConfig.ComposeProjectNamePrefix, "Compose project prefix name should be empty.")
-_, ok = configMap[composeServiceNamePrefixKey]
-assert.True(t, ok, "Compose service name prefix should exist in config.")
-assert.Empty(t, readConfig.ComposeServiceNamePrefix, "Compose service prefix name should be empty.")
-_, ok = configMap[cfnStackNamePrefixKey]
-assert.True(t, ok, "CFNStackNamePrefix should exist in config.")
-assert.Empty(t, readConfig.CFNStackNamePrefix, "CFNStackNamePrefix should be empty.")
+	// Overwrite
+	parser := setupParser(t, dest, false)
+	saveConfigWithCluster(t, parser, dest)
 
+	// Ensure that what has been read is correct
+	// readConfig, err := parser.GetConfigs("", "")
+	//  Temporarily commenting out this code since we need this test cases but can't test it
+	// until the saving configs PR begins
+	// assert.NoError(t, err, "Error reading config")
+	// assert.Equal(t, testClusterName, readConfig.Cluster, "Cluster name mismatch in config.")
+	// assert.Empty(t, readConfig.ComposeProjectNamePrefix, "Compose project prefix name should be empty.")
+	// assert.Empty(t, readConfig.ComposeServiceNamePrefix, "Compose service prefix name should be empty.")
+	// assert.Empty(t, readConfig.CFNStackNamePrefix, "CFNStackNamePrefix should be empty.")
```

Comment:
Go up a few lines:
```
//  Temporarily commenting out this code since we need this test cases but can't test it
// until the saving configs PR begins
```
Until I add the code for saving the new configs this test case will not work; but it is useful. I guess I'll just delete it....